### DISCUSSION TITLE
release 1.0.0

### DISCUSSION THREAD

**Discussion by MrDen256** [EthanArbuckle/carplay-cast#39] [General]
Hi! 
Thanks for the great tweak!
In release 1.0.0, hiding the app via settings does not work. This version was download from oficial repo. There are also no references to the source codes of version 1.0.0. Also version 0.0.1-6 is different here and in the Repo.

**Comment by EthanArbuckle** [comment]
Sorry, I think BigBoss broke it :-/. I’m just going to pull it from that repo. You can still get it here in the meantime
